Behavioral/Specification/README.rst

```diff
@@ -9,6 +9,11 @@ checked against. The composite specification class has one method called
 ``isSatisfiedBy`` that returns either true or false depending on whether
 the given object satisfies the specification.
 
+Sample:
```

Comment:
Right now we also need the modifications in README.md, could you please add it there, too?